src/bindings.c

```diff
@@ -722,7 +728,7 @@ static void __attribute__((constructor)) lxcfs_init(void)
 	int i = 0;
 	pid_t pid;
 
-	lxcfs_info("Running constructor %s to reload liblxcfs with count", __func__);
+	lxcfs_info("Running constructor %s to reload liblxcfs", __func__);
```

Comment:
Ah, fixed here. Mind rebasing the other one to avoid the flip/flop?

Reply:
Yip, will fix.

Reply:
fixed